PAWS/Study Information/EnrollmentGroup.swift

```diff
@@ -0,0 +1,46 @@
+//
+// This source file is part of the PAWS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseAuth
+import FirebaseFirestore
+import Spezi
+import SpeziAccount
+import SpeziFirebaseConfiguration
+import SwiftUI
+
+
+@Observable
+class EnrollmentGroup: Module, EnvironmentAccessible {
+    @ObservationIgnored @Dependency private var configureFirebaseApp: ConfigureFirebaseApp
+    var dateOfBirth: Date?
```

Comment:
We can keep this property private to ensure that a user only accesses the `studyType`
```suggestion
    private var dateOfBirth: Date?
```